eclair-front/src/main/scala/fr/acinq/eclair/Boot.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 ACINQ SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.acinq.eclair
+
+import java.io.File
+
+import akka.actor.ActorSystem
+import com.typesafe.config.{ConfigFactory, ConfigParseOptions, ConfigSyntax}
+import grizzled.slf4j.Logging
+import kamon.Kamon
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+/**
+ * Created by PM on 25/01/2016.
```

Comment:
I know you started working early on this feature, but I don't believe you there :smile:

Reply:
I *may* have copy pasted that one. Removed in aed39363b.